[INFO   ] Logger: Record log in /storage/emulated/0/asistente_predicacion_migracion_kivy/.kivy/logs/kivy_25-05-14_58.txt
[INFO   ] Kivy: v2.3.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.13/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.13.2 (main, Mar 31 2025, 08:14:59) [GCC 11.4.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 195 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2 (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/asistente_predicacion_migracion_kivy/base_screen.py", line 18, in <module>
[WARNING] stderr:     self.layout.add_widget(btn)
[WARNING] stderr:     
[WARNING] stderr: NameError: name 'self' is not defined
